cdap-docs/application-templates/source/etl/custom.rst

```diff
@@ -176,9 +176,9 @@ Example::
   }
 
 
-Creating a Realtime Source Plugin
----------------------------------
-A Realtime Source Plugin can be created from this Maven archetype:
+Creating a Real-time Source Plugin
```

Comment:
Should the capitalized version of real-time be Real-Time? I did some research on hyphenated words in titles, and most seem to agree that if the second word is a noun, it should be capitalized (similar to Co-Founder, or Ex-Wife), whereas if it is a preposition, it should be lower-case (such as Run-off).  To me personally Real-Time seems better than Real-time.